src/components/expanded-state/custom-gas/FeesGweiInput.tsx

```diff
@@ -121,34 +132,38 @@ export default function FeesGweiInput({
   }, [trigger, prevTrigger, actionType, plusAction, minusAction]);
 
   return (
-    <Wrapper>
-      <GweiStepButton
-        buttonColor={buttonColor}
-        onLongPress={onMinusLongPress}
-        onLongPressEnded={onLongPressEnded}
-        onPress={onMinusPress}
-        shouldLongPressHoldPress
-        type={MINUS_ACTION_TYPE}
-      />
-      <GweiInputPill
-        color={buttonColor}
-        editable={editable}
-        onBlur={onBlur}
-        onChange={onChange}
-        onFocus={onInputPress}
-        onPress={onInputPress}
-        ref={inputRef}
-        testID={testID}
-        value={value}
-      />
-      <GweiStepButton
-        buttonColor={buttonColor}
-        onLongPress={onPlusLongPress}
-        onLongPressEnded={onLongPressEnded}
-        onPress={onPlusPress}
-        shouldLongPressHoldPress
-        type={PLUS_ACTION_TYPE}
-      />
-    </Wrapper>
+    <Box marginRight="-5px">
```

Comment:
Not sure I see why we are using negative margins here?

Reply:
it's to achieve margin right of 19 https://www.figma.com/file/PgOKQjlR9uzblIHR3sn6tP/gas-v2?node-id=636%3A1255
the wrapper component has 24 

this is the first input with -5px, so 19px padding right, the second one is aligned to 24 as the rest
![image](https://user-images.githubusercontent.com/12115171/188036563-327a2456-8f4e-4918-a1a9-35bdfd817e84.png)

i moved the margin to the higher component since it was confusing to have it here